### ISSUE TITLE
Deprecation warnings

### ISSUE THREAD

**Issue by mdmeeker** [METHODS-Group/DRDMannTurb#14]
There are several deprecation warnings that we ought to fix up
 - `UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. ...`
 - `UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument.` on `return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]`
 -
